src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/CustomAttributes/CustomAttributeHelpers.cs

```diff
@@ -18,7 +18,7 @@ public static CustomAttributeNamedArgument ToCustomAttributeNamedArgument(this T
             if (members.Length == 0)
                 throw new MissingMemberException(attributeType.FullName, name);
             if (members.Length > 1)
-                throw new AmbiguousMatchException();
+                throw ThrowHelper.GetAmbiguousMatchException(members[0]);
```

Comment:
Maybe I should hoist this one to follow pattern better?

Reply:
better to follow the same pattern